test/TestExpectations.json

```diff
@@ -1236,10 +1236,10 @@
     "expectations": ["SKIP"]
   },
   {
-    "testIdPattern": "[page.spec] Page ExecutionContext.queryObjects should work for non-blank page",
+    "testIdPattern": "[page.spec] Page ExecutionContext.queryObjects should work for non-trivial page",
```

Comment:
Before, this test case was skipped because `Page ExecutionContext.queryObjects should work` is marked as skipped.